src/main/resources/db/postgresql/V1_18__CREATE_TABLE_FILTER_FIELD.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS search_field;
```

Comment:
should this file be included?

Reply:
yes, as discussed it's for flyway consistency when we merge wcs branch back to cs